Plugin.pm

```diff
@@ -1378,6 +1388,34 @@ sub QobuzGetTracks {
 				nextWindow => 'parent'
 			};
 
+			if (my $item = trackInfoMenuBooklet($client, undef, undef, $album)) {
+				push @$items, $item;
+			}
+
+			# Build a consolidated list of all artists on the album
+			my @uniquePerformers;
+			if ( scalar @performers ) {
+				my %seen = ();
+				my $tracks;
+				foreach my $item (@performers) {
+					push @{$tracks->{$item->{'name'}}->{'tracks'}}, $item->{'track'};
+					delete $item->{'track'};
+					push(@uniquePerformers, $item) unless $seen{$item->{'name'}}++;
+				}
+				foreach my $item (@uniquePerformers) {
+					my @tracks = @{$tracks->{$item->{'name'}}->{'tracks'}};
+					if ( @tracks && scalar @tracks < $album->{tracks_count} ) {
+						$item->{'name'} .= scalar @tracks == 1 ? " (" . cstring($client, 'PLUGIN_QOBUZ_TRACK_LC') . " " : " (" . cstring($client, 'PLUGIN_QOBUZ_TRACKS_LC') . " ";
+						$item->{'name'} .= join(", ", @tracks) . ")";
+					}
+				}
+				my $item = {
+					name => cstring($client, 'ALL_ARTISTS'),
```

Comment:
Ok, gave that a try to understand the experience: this is basically a "Credits" menu, isn't it? The "All Artists" string doesn't really work in German, as it's too specific and would be wrong for the engineers, mixers, and what not.

I'm using the following in MAI:

```
	EN	Credits
	FR	Crédits
	NL	Credits
```